src/sot-dynamic-pinocchio.cpp

```diff
@@ -266,12 +267,13 @@ dg::Vector& DynamicPinocchio::getLowerPositionLimits(dg::Vector& res, const int&
       }
       // Found a Spherical Joint.
       // Assuming that spherical joint limits are unset
-      res(fillingIndex) = std::numeric_limits<double>::min();
-      res(fillingIndex + 1) = std::numeric_limits<double>::min();
-      res(fillingIndex + 2) = std::numeric_limits<double>::min();
+      res(fillingIndex) = std::numeric_limits<double>::lowest();
```

Comment:
FYI, this is C++11 only. I don't know whether this is intentional.

Reply:
If you want C++98 compatible code, use -max(). It isn't as good (see numeric_limits doc) but in this context, it is sufficient.

Reply:
Good point thanks. I forgot to check that.